pkg/utils/utils.go

```diff
@@ -329,6 +331,13 @@ func configSriovDevice(iface *sriovnetworkv1.Interface, ifaceStatus *sriovnetwor
 							return err
 						}
 					}
+					if addr == zero_mac_address {
```

Comment:
`addr` here is VF PCI address, you would need to check all zero mac against `vfLink.Attrs().HardwareAddr`.